agent/bench-scripts/test-bin/mock-postprocess/user-benchmark-wrapper

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
I think this mock should be removed, the unit tests until tools-scripts/post-process moved to bench-scripts pbench-user-benchmark.  Can you try that?